CHANGELOG.md

```diff
@@ -9,6 +9,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased] - ReleaseDate
 
+### Changed
+
+- `mc-sgx-tservice::SealedBuilder` utilizes the dedicated error type
+  `mc-sgx-tservice::SealError`. Previously `mc-sgx-tservice::SealedBuilder`
+  utilized `mc-sgx-core-types::Error`.
+- `mc-sgx-tservice::SealedBuilder::new()` and
```

Comment:
Not sure if someone has suggestions for wording to combine these two, or if they should be combined.

Reply:
changed based on review suggestion